4.3/Dockerfile

```diff
@@ -21,9 +22,17 @@ COPY	index.php .
 
 ENV	ADMINER_VERSION 4.3.0
 ENV	ADMINER_DOWNLOAD_SHA256 3584e1b3c7f853118d66a92f9c0996c8db39b05ac2a2b2623ff1676ec09bc08e
+ENV	ADMINER_SRC_DOWNLOAD_SHA256 1e82e9d4766c94a022c1f8e3357e8997ad03b29d2aae171ca2a10490342aee1f
 
 RUN	curl -fsSL https://www.adminer.org/static/download/$ADMINER_VERSION/adminer-$ADMINER_VERSION-en.php -o adminer.php \
-&&	echo "$ADMINER_DOWNLOAD_SHA256  adminer.php" |sha256sum -c -
+&&	echo "$ADMINER_DOWNLOAD_SHA256  adminer.php" |sha256sum -c - \
+&&	curl -fsSL https://github.com/vrana/adminer/archive/v$ADMINER_VERSION.tar.gz -o source.tar.gz \
+&&	echo "$ADMINER_SRC_DOWNLOAD_SHA256  source.tar.gz" |sha256sum -c - \
+&&	tar xzf source.tar.gz -C . --strip-components=1 "adminer-$ADMINER_VERSION/designs/" \
```

Comment:
I'm fairly sure `-C .` is redundant, because it's the implicit default. Please check and remove if I'm correct.